### PR TITLE
Added multiple ConversionException and UnknownDecoderException

### PR DESCRIPTION
**Pull request by StephanThierry**
(Now including minor fixes from test on 43k+ video files)

Added multiple ConversionException and UnknownDecoderException from Ffmpeg output messages.

This is to separate invalid videos (ConversionException ) from videos that are simply not supported by Ffmpeg (UnknownDecoderException)

Changed incorrect summary in: UknownDecoder.cs and filename to UnknownDecoder.cs

Added extra info to ArgumentException. (There is a bug where Duration is sometimes incorrect in Conversion tasks)

### CODE REVIEW

File: Xabe.FFmpeg/FFmpegWrapper.cs [hunk 1]
```diff
@@ -106,7 +106,17 @@ internal class FFmpegWrapper : FFmpeg
                             return false;
                         }
 
-                        if (_outputLog.Any(x => x.Contains("Unknown decoder")))
+                        if (_outputLog.Any(x => x.Contains("Invalid NAL unit size")))
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/FFmpegWrapper.cs]: Please extract all these strings to const array and check in loop if output string contains any of these errors.

**Comment by 06needhamt** [comment]
@tomaszzmuda it seems like the requested work has been done. Can this be re-reviewed?

**Comment by tomaszzmuda** [comment]
@06needhamt 
I'm going to change this a bit and merge (but probably in different PR).
I totally forgot to do this, sorry!

**Comment by 06needhamt** [comment]
@tomaszzmuda Great. I’m keen to see this feature merged as it will simplify diagnosing issues, but no pressure.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by tomaszzmuda** [comment]
Merged in #224

**Pull request closed without merging**